test-integration/task-plan/publish.coffee

```diff
@@ -6,29 +6,38 @@ TEACHER_USERNAME = 'teacher01'
 
 describe 'Assignment Publishing Tests', ->
 
-  @it 'Publishes a Reading with opensAt to tomorrow and deletes (idempotent)', ->
-    title = @freshId()
-
-    @login(TEACHER_USERNAME)
+  beforeEach ->
+    @title = freshId()
+    new User(@, TEACHER_USERNAME).login()
```

Comment:
Yes, I considered that but decided against it because the User will have methods other than login at some point, like `displayProfile` or `selectMenuOption`.  When that happens then the username would need to be supplied as part of the constructor.